index.html

```diff
@@ -1,44 +1,224 @@
-        <i class="fa-solid fa-bars"></i>
-     </nav>
-    </header>
-    <section class="container">
-        <div class="row">
-                <h1>Hey There.
-                  I’m Alex</h1>
-                <h3>I’am a Software Developer</h3>
-                <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
-                  don’t hesitate to contact me. 
-                </p>
-        </div>
-        <section class="media-icons">
-            <ul class="icons">
-             <li><a href="#"><i class="fa-brands fa-github"></i></a></li>
-             <li><a href="#"><i class="fa-brands fa-linkedin"></i></a></li>
-             <li><a href="#"><i class="fa-brands fa-angellist"></i></a></li>
-             <li><a href="#"><i class="fa-brands fa-twitter-square"></i></a></li>
-             <li><a href="#"><i class="fa fa-medium"></i></a></li>
-          </ul>
-      </section>
+  <header>
+    <nav class="header-toolbar">
+      <h4>emmanuel moombe</h4>
+      <i class="fa-solid fa-bars"></i>
+    </nav>
+  </header>
+  <section class="container">
+    <div class="row">
+      <h1>Hey There. I’m Emmanuel</h1>
+      <h2>I’am a Software Developer</h2>
+      <p>
+        I can help you build a product , feature or website Look through some
+        of my work and experience! If you like what you see and have a project
+        you need coded, don’t hesitate to contact me.
+      </p>
+    </div>
+    <section class="media-icons">
+      <ul class="icons">
+        <li>
+          <a href="#"><i class="fa-brands fa-github"></i></a>
+        </li>
+        <li>
+          <a href="#"><i class="fa-brands fa-linkedin"></i></a>
+        </li>
+        <li>
+          <a href="#"><i class="fa-brands fa-angellist"></i></a>
+        </li>
+        <li>
+          <a href="#"><i class="fa-brands fa-twitter-square"></i></a>
+        </li>
+        <li>
+          <a href="#"><i class="fa fa-medium"></i></a>
+        </li>
+      </ul>
+    </section>
+  </section>
+  <section class="work-section">
+    <h3 class="title1">My Recent Works</h3>
+    <hr class="separator" />
+    <div class="card-container">
+      <img src="images/Placeholder1.png" class="intro-img" alt="img" />
+      <h6 class="card-title">Multi-Post Stories</h6>
+      <article class="para-discrp1">
+        <p class="para-sec1">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+      </article>
+      <div class="btn-skill">
+        <ul class="skill-list">
+          <li class="list-item-1">css</li>
+          <li class="list-item-1">html</li>
+          <li class="list-items-1">bootstrap</li>
+          <li class="list-item-1">Ruby</li>
+        </ul>
```

Comment:
- [ ] Kindly try to match the Figma design for spacing between each technology to match the Figma design. The space between each technology is exactly the same. The last technology in your design has an extra space than the other three technologies.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/159468865-b72bace4-68e6-44a1-8d33-19cb3cf40715.png) | ![image](https://user-images.githubusercontent.com/24703171/159468952-ac292cf3-5533-43fd-aeb2-de5184f5f119.png) |